.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: CI/CD release pipeline
+
+# Run this workflow every time a new commit pushed to your repository
+on: push
+
+jobs:
+  tox-test:
+    name: Tox test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Run Tox tests
```

Comment:
Is it beneficial to run it on Fedora? Pulls of `fedorapython/fedora-python-tox` are taking around 25s.

Reply:
This image has a pre-installed tox. Do you have any alternative?